google-embed/google-embed-package/editorextensions/drive/src/picker/googledrivepickermodal.ts

```diff
@@ -0,0 +1,55 @@
+import {
+    arrayValidator,
+    EditorClient,
+    either,
+    GuardToType,
+    isString,
+    Modal,
+    objectValidator,
+    someValue,
+} from 'lucid-extension-sdk';
+import content from '../../resources/googledrivepicker.html';
+import { getGoogleDriveLocale } from '../i18n/googledrivelocale';
+
+export type PickerResult = GuardToType<typeof isPickerResult>;
+export const isPickerResult = either(
+    objectValidator({ 'action': someValue('cancel', 'error') }),
+    objectValidator({ 'action': someValue('picked'), 'docUrls': arrayValidator(isString) }),
+);
+
+export class GoogleDrivePickerModal extends Modal {
+    constructor(client: EditorClient, private readonly accessToken: string) {
+        super(client, { content, chromeless: true, fullScreen: true, transparentBackground: true });
```

Comment:
Instead of using content, we recently supported static file serving, so [this]( https://developer.lucid.co/extension-api/#specify-content) is the best way to link to a static html now. I think it should be a pretty fast change.

Reply:
Discussed offline. We'll make this change in another PR as it wasn't working straight out of the gate.